docs/docs/dev/networking/cni.md

```diff
@@ -9,6 +9,10 @@ There are two options to do this with Kuma:
 
 Kuma CNI can be leveraged in the two installation methods for Kubernetes: using [`kumactl`](../../installation/kubernetes) and with [Helm](../../installation/helm). The default settings are tuned for OpenShift with Multus, therefore to use it in other environments we need to set the relevant configuration parameters.
 
+:::warning
+Kuma CNI applies NetworkAttachmentDefinition(NAD) to applications in a namespace with `kuma.io/sidecar-injection` label or annotation. To correctly apply NAD to the applications not in a Mesh, add the label or annotation `kuma.io/sidecar-injection` with the value `false` or `disabled` to the namespace.
```

Comment:
```
Kuma CNI applies NetworkAttachmentDefinition(NAD) to applications in a namespace with `kuma.io/sidecar-injection` label or annotation. To correctly apply NAD to the applications not in a Mesh, add the label or annotation `kuma.io/sidecar-injection` with the value `false` or `disabled` to the namespace.
```
how about ->
```
Kuma CNI applies NetworkAttachmentDefinition(NAD) to applications in a namespace with `kuma.io/sidecar-injection` label. To apply NAD to the applications not in a Mesh, add the label `kuma.io/sidecar-injection` with the value `disabled` to the namespace.
```

1) annotation is deprecated anyways. Skip it.
2) ~correctly~ = too wordy
3) `false` or `disabled`, let's teach a user one way of doing things.